docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+field at a time.
+
+**Note:** Ingress IOCs cannot be edited in the Carbon Black Cloud console UI at this time, due to a UI limitation
+on editing two properties of an IOC at the same time.
+
+You *can* include more than one entry (query or match element) in an individual IOC, but in order to ignore or disable
+one of those entries, you would either have to edit the IOC or disable it entirely (thus disabling *all* entries in
+that IOC).  It is recommended to use only one entry per IOC, for ease of management, unless you have already vetted the
+entries and don't expect to have to disable them individually.
+
+Both IOCs and reports may include a ``link`` property, which is used by the Carbon Black Cloud console UI as a hint
+to indicate that this IOC or report is being managed outside of the console.  If this property is not ``None``,
+the console UI will disable the ability to edit the IOC or report, but they can still be edited via the API.
+
+Tips for Using IOCs
++++++++++++++++++++
+
+* You can safely ignore certain fields in an IOC.  For example, fields like ``alert_id`` and ``process_guid`` will
+  always uniquely identify just a single record in your organization's data, whereas a field like ``org_id`` will be
+  a constant across *all* your organization's data.
+* Timestamp fields such as ``backend_timestamp`` are useful in ad-hoc queries, to look for data occurring before or
+  after a certain date, but are of limited usefulness over the span of time a watchlist may be running.
+* A list of hashes (such as with ``process_sha256``) can be of limited value.  They are inconvenient to keep current,
+  especially as software (whether legitimate or malicious) gets updated over time, but are definitely easier to manage
+  with ``equality`` IOCs.
+* Counter fields (such as ``netconn_count``) can be useful with range queries to locate processes that are using a
+  large number of resources.  For example, the query ``netconn_count:[500 TO *]`` will match only processes that make
+  a large number of network connections.
+* When using ingress IOCs, be careful of errant characters in the ``values`` list, such as leading or trailing
+  whitespace or embedded newline characters.  These errant characters may cause the IOCs to fail, leading to false
+  negative results.
+* ``equality`` IOCs for IPv4 fields (e.g. ``netconn_remote_ipv4``) cannot support CIDR notation; full IP addresses
+  must be used.
+* ``equality`` IOCs for IPv6 fields (e.g. ``netconn_remote_ipv6``) do not support standard or CIDR notation at this
+  time. All IPv6 addresses must omit colon characters, spell out all zeroes in the address, and represent all
+  alphabetic characters in uppercase. For example, "ff02::fb" becomes "FF0200000000000000000000000000FB".
+* Query IOCs must always use field-prefixed queries (key-value pairs); they do not support just searching for a value
```

Comment:
This would be worth while to mention in the above section where we describe query based IOCs

Reply:
This is part of a subsection called "Tips for Using IOCs." We do not differentiate between query and ingress IOCs in this section. Unless you think it should be split into two sections.  But some of the tips are applicable to both types.

Reply:
I would think this should be mention the paragraph 

```
*Query IOCs* are those with a ``match_type`` of ``query``; their ``values_list`` contains a single string that specifies a query compatible with process searches.
```

Reply:
I've separated two bullet points and brought them up where you suggest.